README.md

```diff
@@ -55,10 +55,9 @@ PS: A Login screen is implemented in both flows. It's just a template that you c
 
 ```
 npm install
-npm run build
 ```
 
-- Follow the steps in the [Quick Start: Lightning Web Components](https://trailhead.salesforce.com/content/learn/projects/quick-start-lightning-web-components/) Trailhead project. You will:
+- Follow the steps to setup your developement environment, for example in the [Quick Start: Lightning Web Components](https://trailhead.salesforce.com/content/learn/projects/quick-start-lightning-web-components/) Trailhead project. You will:
```

Comment:
It's not you must though. But I can change the rest of the sentence sure.
The sentence says follow this tutorial and it will run you through these steps, not you must do these steps.